.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+Thanks for wanting to fix something on Sequelize - we already love you!
+Please fill in the template below.
+If unsure about something, just do as best as you're able.
+-->
+
+## Pull Request Checklist
+
+<!-- Please make sure to review and check all of these items: -->
+
+- [ ] Is this also happen in the latest version of Sequelize website?
+- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
+
+<!-- NOTE: these things are not required to open a PR and can be done afterwards / while the PR is open. -->
+
+## Description Of Change
+
+<!-- Please provide a description of the change here. -->
+
+## Todos
```

Comment:
I propose to remove this, it just adds noise to PRs. Users that make incomplete PRs (usually us) can always just add it back in their PR